clients/cpp/zawgyi_detector.cpp

```diff
@@ -275,3 +275,13 @@ double ZawgyiDetector::GetZawgyiProbability(const char* input_utf8,
                                             int32_t length) const {
   return model_->Predict(input_utf8, length);
 }
+
+// C bindings (declared with extern "C").
+double GMTGetZawgyiProbability(const char* input_utf8) {
+  return GMTGetZawgyiProbabilityWithLength(input_utf8, -1);
+}
+
+double GMTGetZawgyiProbabilityWithLength(const char* input_utf8, int32_t length) {
+  ZawgyiDetector detector;
```

Comment:
Suggestion (can ignore if too hard): The constructor does a bit of work (parsing the model).  It would be nicer if you didn't have to construct the ZawgyiDetector each time.  The pattern ICU uses for this is an opaque struct that you open and close, like this:

```c
GMTZawgyiDetector detector = gmt_openZawgyiDetector();
double result = gmt_getZawgyiProbability(detector, "...");
gmt_closeZawgyiDetector(detector);
```

Reply:
Done, thanks!